# Idea: The new bus route improves compute thus new compute time is less than old compute time.
$$H0: T1>= T0$$
$$Ha: T1<T0$$
where T0 is old compute time and T1 is new compute time.

significance level: 0.05

In [21]:
import numpy as np
import pandas as pd
import math

In [23]:
test = pd.read_csv("assign3_test.csv")
test.head()

value
0  31.622239
1  32.821376
2  30.229101
3  31.413766
4  39.010550

In [28]:
mean = test.mean()
p_mean = 36
sd = test.std()
N = len(test)

# Calculates z value

In [29]:
z = (p_mean - mean)/(sd/math.sqrt(N))
z

value    2.149052
dtype: float64

# The z value of 2.149052 illustrates the number of standard deviations away the sample mean is from the population mean.
# As calculated, test z score = 2.149052 > 1.96, test score rejects null hypothesis at a 0.05 significance level. 